### DISCUSSION TITLE
Borders around unselected menu items?

### DISCUSSION THREAD

**Discussion by roryyamm** [tsujan/Kvantum#833] [Q&A] [answered]
I'm trying to edit an existing theme, QTStepWMakerDefault, to have menus more akin to WindowMaker's, but for the life of me, I can't seem to figure it out. The issue is in the menus - WMaker (And by extension, NextStep/OpenStep) has borders around every item in a menu:
![Screenshot From_2022-11-06 124545](https://user-images.githubusercontent.com/24758288/200186446-4b9f54df-3f03-4233-bbd6-585bc1e6dc8e.png)
However, through playing around with interior.element, indicator.element, and frame.element for the various widgets, I've only managed to get the intended behavior when hovering over:
![Screenshot From_2022-11-06 125052](https://user-images.githubusercontent.com/24758288/200186743-669c32bc-8e85-4730-af0c-7d2ea9f2a85d.png)
Is there a way to achieve this with KVantum? I will admit that I've only been randomly poking and prodding at this point.

**Comment by tsujan** [comment, marked as answer]
> Is there a way to achieve this with KVantum?

No. Kvantum doesn't draw any panel/border for normal/disabled menu-items; they should be hovered or pressed to have panels. I'm afraid you can't copy the appearance of WindowMaker's menus with Kvantum.

**Reply by roryyamm** [reply]
…Shit. 
Still, thank you for the quick response to this question.

**Reply by tsujan** [reply]
You're welcome.

Kvantum is flexible, but not *infinitely* flexible ;)
